auth/tests/api/helpers.rs

```diff
@@ -117,6 +120,20 @@ impl TestApp {
         self.send_request(request).await
     }
 
+    pub async fn get_jwt_from_non_admin_api_key(
```

Comment:
Should we refactor or remove the original "get_jwt_...` method, since it's no longer usable?

Reply:
Thanks! Simplified the jwt from api key generation too. LMK if it looks better now.